clickfile.py

```diff
@@ -1253,10 +1255,44 @@ def build(tag):
 @k6.command("run", help="Run k6 performance test")
 @click.option("-n", "--network", default="local", help="Which network to use for envs assignment")
 @click.option("-s", "--script", default="./loadtesting/k6/tests/sendNeon.test.js", help="Path to k6 script")
+@click.option("-u", "--users", default=None, help="Number of users (will be generated before load test run)")
+@click.option("-b", "--balance", default=None, help="Initial balance of each user in Neon")
+@click.option("-a", "--bank_account", default=None, help="Bank account address")
 @catch_traceback
-def run(network, script):
-    with open('envs.json') as json_file:
+def run(network, script, users, balance, bank_account):
+    config = k6_set_envs(network, "envs.json", users, balance, bank_account)
+    
+    print("Creating accounts...")
+    accounts = {}
+    web3_client = web3client.NeonChainWeb3Client(proxy_url=config[network]['proxy_url'])
+    faucet = Faucet(faucet_url=config[network]['faucet_url'], web3_client=web3_client)
+    account_manager = EthAccounts(web3_client, faucet, None)
+
+    # We create 2 accounts for each k6 virtual user: sender and receiver.
+    # We need to create 2*VU accounts cause we want to make sure account's nonces is not overlapping.
+    for i in range(2*int(users)):
+        account_sender = account_manager.create_account(balance=int(balance))
+        account_receiver = account_manager.create_account(balance=0)
+        print(f"Account {str(i)} sender: {account_sender.address}, initial balance: {balance} Neon.")
+        print(f"Account {str(i)} receiver: {account_receiver.address}")
+        accounts[i] = {"sender_address": str(account_sender.address), 
+                       "sender_key": str(account_sender.key.hex())[2:], 
+                       "receiver_address": str(account_receiver.address)}
+
+    with open('./loadtesting/k6/data/accounts.json', 'w', encoding='utf-8') as f:
```

Comment:
lets move it from clickfile.py to another script  to have here only run command 
because except accounts we need deployed contracts and maybe some another prepared data in future
It is better not to overload clickfile

Reply:
agree

Reply:
moved to utils/k6_helpers.py